Atomic/scan.py

```diff
@@ -87,6 +87,10 @@ def set_scanner():
             raise ValueError(yaml_error)
 
         self.results_dir = os.path.join(self.results, self.scanner, self.cur_time)
+        last_scanned_file = os.path.join(self.results, "last_scanned.json")
+        with open(last_scanned_file, 'w') as f:
+            json.dump(self.cur_time, f, indent=4)
+
 
```

Comment:
This was one approach I thought of. Another involves comparing different timedate objects which I was looking into. In the case of Andreas' design, it has a box which lists how many containers and images are vulnerable and then lists the time of the last scan.